certs/manager.go

```diff
@@ -596,8 +596,16 @@ func (c *CertificateManager) ValidateRequestCertificate(certIDs []string, r *htt
 
 	certID := HexSHA256(leaf.Raw)
 	for _, cert := range c.List(certIDs, CertificatePublic) {
+		// In case a cert can't be parsed and is invalid,
```

Comment:
What if we remove here and rephrase `We can't validate this, continue` as `nil cert means invalid, check others` and remove `?

Reply:
I'd leave the comment. We can clear it in List() alternatively to skip the nil entry, but other code may depend on it. Similar nil checks exist in /gateway, but as those are descendant of List(), the comment is more applicable in this part of the code.

Reply:
Text wise, if you find any sentence clearer, we can rephrase of course.